API/chat/tests.py

```diff
@@ -47,15 +54,22 @@ def setUp(self):
         super(ChatTests, self).setUp()
         self.channel = G(Channel, name='Channel')
 
+    def privileged_operation(self, endpoint, data, method):
+        return getattr(self.client, method)(
+            endpoint,
+            data,
+            HTTP_AUTHORIZATION=settings.PASS
+        )
 
 class MessageViewPOSTTests(ChatTests):
     def post_and_get_response(self, text, timestamp, username, typing):
         """
         Posts a message on chat:message and returns the response
         """
-        return self.client.post(
+        return self.privileged_operation(
```

Comment:
Why do you need privileged operations on this commit, since you haven't use the privileged decorator on the views?